documentation/reference/ci_cd.md

```diff
@@ -175,8 +175,8 @@ published to GHCR. This job determines those images.
 | Change             | `upstream_db` | `catalog` | `ingestion_server` | `api` | `api_nginx` | `frontend` |
 | ------------------ | ------------- | --------- | ------------------ | ----- | ----------- | ---------- |
 | `catalog`          | 🛠️            | 🚀        |                    |       |             |            |
-| `ingestion_server` | 🛠             |           | 🚀                 | 🛠     |             |            |
-| `api`              | 🛠             |           | 🛠                  | 🚀    | 🚀          |            |
+| `ingestion_server` | 🛠️            |           | 🚀                 | 🛠️    |             |            |
+| `api`              | 🛠️            |           | 🛠️                 | 🚀    | 🚀          |            |
 | `frontend`         |               |           |                    |       |             | 🚀         |
 
 🚀 implies that the image is published to GHCR. 🛠️ implies that the image is
```

Comment:
Does the hammer and wrench in this line have color?

Reply:
Yup! I changed all the ones that were different.

![image](https://user-images.githubusercontent.com/24264157/232670146-21d010fe-6154-4da0-8f40-2d117cb5406b.png)